angular/src/directives/control-value-accessors/numeric-value-accessor.ts

```diff
@@ -18,13 +18,13 @@ export class NumericValueAccessorDirective extends ValueAccessor {
     super(injector, el);
   }
 
-  @HostListener('ionChange', ['$event.target'])
-  _handleIonChange(el: any): void {
-    this.handleChangeEvent(el, el.value);
+  @HostListener('ionInput', ['$event.target'])
+  handleInputEvent(el: HTMLIonInputElement): void {
+    this.commitValueChange(el, el.value);
```

Comment:
The naming of `commitValueChange` seems odd here. In the core implementation, we consider "committing the value" to be when `change` fires. In this file, we consider committing to be when `input` (and therefore `ionInput`) fires.

I'm assuming this is so that form validation is re-run whenever `ionInput` fires? If so, maybe a better name would be `handleInputEvent`?

Reply:
The only reason I would suggest not renaming this function to `handleInputEvent` at this stage, is this base class is shared across all the value accessors. Only `ion-input` is updated to use `ionInput` instead of `ionChange`. All other value accessors currently use `ionChange`. 

Does it make sense to update that API now, knowing they will all eventually use `ionInput` or similar or should that happen once the final component is migrated and all components use the input event?

Reply:
As long as we update this to be consistent before the feature launches, that should be fine I think

Reply:
Thinking about this more, not all value accessors will bind to `ionInput` (that event is only available on a subset of our components). 

The fact that this wraps Angular's `onChange` method implementation for value accessors, should we call it: `handleChange` or `handleValueChange` instead?